packages/cli/src/cli/basemaps/tileset.config.ts

```diff
@@ -0,0 +1,154 @@
+import { EpsgCode } from '@basemaps/geo';
+import * as z from 'zod';
+import { parseRgba } from './tileset.util';
+
+export const ImageryConfigDefaults = {
+    priority: 1000,
+    minZoom: 0,
+    maxZoom: 32,
+};
+
+const zZoom = z.number().refine((val) => val >= ImageryConfigDefaults.minZoom && val <= ImageryConfigDefaults.maxZoom, {
+    message: `must be between ${ImageryConfigDefaults.minZoom} and ${ImageryConfigDefaults.maxZoom}`,
+});
+
+const zImageryRule = z
+    .object({
+        priority: z
+            .number()
+            .refine((val) => val >= -1 && val <= 10000, { message: 'must be between -1 and 10000' })
```

Comment:
What does a priority of `-1` mean?

Reply:
-1 is turn off as it is in the other tile metadata commands

Reply:
Do we not delete it by just removing it from the config file?